includes/admin/class-wc-stripe-payment-requests-controller.php

```diff
@@ -39,6 +39,18 @@ public function admin_scripts() {
 		);
 		wp_enqueue_script( 'wc-stripe-payment-request-settings' );
 
+		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
+		$params = [
```

Comment:
```suggestion
		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
		$params          = [
```

Small whitespace nit 😅